src/Init/Data/Int/Order.lean

```diff
@@ -584,7 +586,10 @@ theorem add_one_le_iff {a b : Int} : a + 1 ≤ b ↔ a < b := .rfl
 theorem lt_add_one_iff {a b : Int} : a < b + 1 ↔ a ≤ b := Int.add_le_add_iff_right _
 
 @[simp] theorem succ_ofNat_pos (n : Nat) : 0 < (n : Int) + 1 :=
-  lt_add_one_iff.2 (ofNat_zero_le _)
+  lt_add_one_iff.mpr (ofNat_zero_le _)
+
+theorem ofNat_not_neg (n : Nat) : ¬((n : Int) < 0) :=
```

Comment:
Should this be called
```suggestion
theorem not_ofNat_neg (n : Nat) : ¬((n : Int) < 0) :=
```

Reply:
Sure. Done. (It's apparently unused, in any case.)